java/drivers/driver-hazelcast4plus/conf/worker.sh

```diff
@@ -7,20 +7,112 @@
 # See the end of this file for examples for different profilers.
 #
 
-# automatic exit on script failure
-set -e
+# Enable strict error handling
+set -euo pipefail
+
 # printing the command being executed (useful for debugging)
 #set -x
 
-# redirecting output/error to the right log files
-exec > worker.out
-exec 2> worker.err
+# Redirecting output and error to log files with timestamps
+exec > >(tee -a worker.out) 2> >(tee -a worker.err >&2)
+
+log() {
+    local level="$1"
+    shift
+    echo "$(date '+%Y-%m-%d %H:%M:%S') [$level] $*"
+}
 
 # Read the parameters file and add it to the environment
 while IFS='=' read -r key value; do
     export "$key"="$value"
 done < "parameters"
 
+validate_mount_params() {
```

Comment:
Might be nice to a bit more validation beyond checking these are set - validating they're non-empty, correct syntax, etc.

Reply:
Nice suggestion, see 1b58cae